### ISSUE TITLE
r-universe

### ISSUE THREAD

**Issue by dipterix** [dfsp-spirit/drat#1]
Hi @dfsp-spirit , I found [r-universe](https://r-universe.dev/builds/) pretty handy. You don't have to compile your own drat repo in order to get cross-platform binary packages. Hopefully this helps. (I'm not from r-universe trying to advertise

**Comment by dfsp-spirit** [comment]
Thanks, I didn't know about that. I just wanted a place to host packages without CRANs 5 MB limit, in a way that would still allow me to specify them as dependencies of other packages. I'll have a look at it!

**Comment by dipterix** [comment]
Yeah, I saw your regfusionr package. Amazing work! I do think you can get your package on CRAN. You can let users download the data in R command. For example 
https://github.com/dipterix/threeBrain/blob/8b750c09985e954be939ac305b01ab2c8afa8d9f/R/loaders.R#L25-L29

The key is you can store large data via `tools::R_user_dir` (for example https://github.com/dipterix/threeBrain/blob/f561339b4f5480ecc9caecb82850c0eabdc573e7/R/utils.R#L843) to keep package small.

Also, I'm doing CT MRI coregistration these days. Do you have any idea any R package is doing that natively?

**Comment by dfsp-spirit** [comment]
Yeah, I know about R_user_dir and the option to have users download things from remote servers. I am doing that already for fsbrain. It just adds extra complexity and is annoying, it sometimes brakes the unit tests on CI when a server has a hickup, on some older MacOS systems we have, https downloads for some servers fail because they have outdated SSL certificate stores leading to curl errors when trying to validate the download servers, etc. I am not at all a big fan of that approach I gotta say.

For the CT registration: sorry, no. So far I haven't done any 3D registration stuff in R.

**Comment by dipterix** [comment]
Wow, good to know!
